dhis-2/dhis-services/dhis-service-field-filtering/src/main/java/org/hisp/dhis/fieldfiltering/FieldPathHelper.java

```diff
@@ -204,18 +208,29 @@ public void applyPresets( List<FieldPath> presets, Map<String, FieldPath> fieldP
             {
                 continue;
             }
-
-            if ( PRESET_ALL.equals( preset.getName() ) )
+            if ( FieldPreset.ALL.equals( preset.getName() ) )
```

Comment:
This starts to look like `switch( preset.getName()) {` is the better approach here.

Reply:
Could be yes. Just followed the existing code, think I leave it for now.